docs/content/en/sentry/options.md

```diff
@@ -0,0 +1,251 @@
+  - **Note**: Consider using `hidden-source-map` instead. For most people, that should be a better option but due to it being a breaking change, it won't be set as the default until next major release
+
+### attachCommits
+- Type: `Boolean`
+  - Default: `process.env.SENTRY_AUTO_ATTACH_COMMITS || false`
+  - Only has effect when `publishRelease = true`
+
+### repo
+- Type: `String`
+  - Default: `process.env.SENTRY_RELEASE_REPO || ''`
+  - Only has effect when `publishRelease = true && attachCommits = true`
+
+### disableServerRelease
+- Type: `Boolean`
+  - Default: `process.env.SENTRY_DISABLE_SERVER_RELEASE || false`
+  - See https://docs.sentry.io/workflow/releases for more information
+
+### disableClientRelease
+- Type: `Boolean`
+  - Default: `process.env.SENTRY_DISABLE_CLIENT_RELEASE || false`
+  - See https://docs.sentry.io/workflow/releases for more information
+
+### clientIntegrations
+- Type: `Dictionary`
+  - Default:
+  ``` js
+   {
+      Dedupe: {},
+      ExtraErrorData: {},
+      ReportingObserver: {},
+      RewriteFrames: {},
+      Vue: {attachProps: true, logErrors: this.options.dev}
+   }
+  ```
+  - Sentry by default also enables these browser integrations: `InboundFilters`, `FunctionToString`, `TryCatch`, `Breadcrumbs`, `GlobalHandlers`, `LinkedErrors`, `UserAgent`. Their options can be overridden by specifying them manually in the object.
+  - See https://docs.sentry.io/platforms/javascript/configuration/integrations/default/ and  https://docs.sentry.io/platforms/javascript/configuration/integrations/plugin/ for more information on configuring integrations
+
+### serverIntegrations
+- Type: `Dictionary`
+  - Default:
+  ``` js
+    {
+      Dedupe: {},
+      ExtraErrorData: {},
+      RewriteFrames: {},
+      Transaction: {}
+    }
+  ```
+  - See https://docs.sentry.io/platforms/node/pluggable-integrations/ for more information
+
+### tracing
+- Type: `Boolean` or `Object`
+  - Default: `false`
+  - Enables the BrowserTracing integration for client performance monitoring
+  - Takes the following object configuration format (default values shown):
+  ```js
+    {
+      tracesSampleRate: 1.0,
+      vueOptions: {
+        tracing: true,
+        tracingOptions: {
+          hooks: [ 'mount', 'update' ],
+          timeout: 2000,
+          trackComponents: true
+        }
+      },
+      browserOptions: {}
+    }
+  ```
+  - Sentry docs strongly recommend reducing the `tracesSampleRate` value; it should be between 0.0 and 1.0 (percentage of requests to capture)
+  - The `vueOptions` are passed to the `Vue` integration, see https://docs.sentry.io/platforms/javascript/guides/vue/#monitor-performance for more information
+  - `browserOptions` are passed to the `BrowserTracing` integration, see https://github.com/getsentry/sentry-javascript/tree/master/packages/tracing for more information
+  - `@sentry/tracing` should be installed manually when using this option (it is currently a dependency of `@sentry/node`)
```

Comment:
Can we move it before the "Type" and make it an `<alert>` element? I feel like this should be more exposed.